servers/zms/src/main/resources/messages/principal-review.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+<head>
+    <meta charset="utf-8">
+    <title></title>
+    <style></style>
+</head>
+<body style="margin: 0; background: rgba(231, 232, 231, 1.0);">
+<div class="athenz-wrapper">
+    <div class="mbrapproval">
+        <div class="logo">
+            <img src="cid:logo" class="athenzlogowhite" alt="Athenz logo"/>
+        </div>
+        <div class="hdr">Principal Review Details</div>
+        <div class="bt">Review date for principal <b>{0}</b> in the following roles will expire soon:</div>
```

Comment:
it should say - "You are required to review your <b>{0}</b> principal's membership in the listed roles:"